spring-core/src/test/java/facade/BookingFacadeImplTest.java

```diff
@@ -0,0 +1,154 @@
+package facade;
+
+import model.Event;
+import model.Ticket;
+import model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.List;
+import java.util.Objects;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(locations = "classpath:beans.xml")
+public class BookingFacadeImplTest {
```

Comment:
For naming test classes should be without suffix Impl.

But if you have 2 instance of objects from interface, then yes, you need create test classes with full name